src/rred_reports/reports/tables.py

```diff
@@ -0,0 +1,210 @@
+    "month6_bas_result",
+    "exit_outcome",
+]
+
+
+def school_filter(whole_dataframe: pd.DataFrame, school_id):
+    """Function to filter school"""
+    return whole_dataframe[whole_dataframe.school_id == school_id]
+
+
+# specific filter for each table
+def filter_one(school_dataframe: pd.DataFrame):
+    """Filter for table one, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_two(school_dataframe: pd.DataFrame):
+    """Filter for table two, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_three(school_dataframe: pd.DataFrame):
+    """Filter for table three, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_four(school_dataframe: pd.DataFrame):
+    """Filter for table four, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_five(school_dataframe: pd.DataFrame):
+    """Filter for table five, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def filter_six(school_dataframe: pd.DataFrame):
+    """Filter for table six, for later use
+
+    Args: school_filter(pd.DataFrame)"""
+    return school_dataframe
+
+
+def summary_table_function(school_df: pd.DataFrame):
+    """
+    Args:
+        school_filter(pd.DataFrame)
+
+    Returns:
+        table with the following columns
+            Number of RR teachers
+            Number of pupils served
+            (Pupil outcomes) Discontinued
+            (Pupil outcomes) Referred to school
+            (Pupil outcomes) Incomplete
+            (Pupil outcomes) Left School
+            (Pupil outcomes) Ongoining
+    """
+    columns_used = ["rred_user_id", "pupil_no", "exit_outcome"]
+
+    def filter_summary_table(school_df):
+        """Filter for summary table
+
+        Args: pd.DataFrame
+        """
+        return school_df
+
+    def get_outcome_from_summary(school_df, outcome_type):
+        try:
+            return school_df["exit_outcome"].value_counts()[outcome_type]
+        except KeyError:
+            return 0
+
+    filtered = filter_summary_table(school_df)
+    summary_table = filtered[columns_used]
+
+    return pd.DataFrame(
+        {
+            "number_of_rr_teachers": [summary_table["rred_user_id"].nunique()],
+            "number_of_pupils_served": [summary_table["pupil_no"].nunique()],
+            "po_discontinud": get_outcome_from_summary(summary_table, "Discontinued"),
+            "po_referred_to_school": get_outcome_from_summary(summary_table, "Referred to school"),
+            "po_incomplete": get_outcome_from_summary(summary_table, "Incomplete"),
+            "po_left_school": get_outcome_from_summary(summary_table, "Left school"),
+            "po_ongoing": get_outcome_from_summary(summary_table, "Ongoing"),
+        }
+    )
+
+
+def make_table(school_df: pd.DataFrame, template_path: Path):

```

Comment:
Might be worth renaming this function as I don't think it makes a table. maybe `populate_school_tables`? Also worth parameterising the function so that we can specify the filename that we're writing to